test/pkg/integration/setup.go

```diff
@@ -29,9 +30,40 @@ func SkipWithoutUsername(t *testing.T, username string) {
 		t.Skip("Skipping because requires a username")
 	}
 }
-func Setup(ctx context.Context) (string, string, env.Environment) {
+
+func SkipWithoutEnterpriseLicense(t *testing.T, enterpise bool) {
+	if !enterpise {
+		t.Skip("Skipping because requires enterprise license")
+	}
+}
+
+func EnsureUserExists(t *testing.T, username string, api *ComponentAPI) string {
+	if username == "" {
+		t.Logf("no username provided, creating temporary one")
+		rand.Seed(time.Now().UnixNano())
+		randN := rand.Intn(1000)
+		new_user := fmt.Sprintf("johndoe%d", randN)
```

Comment:
Is this more like it?
```suggestion
		newUser := fmt.Sprintf("johndoe%d", randN)
```